testbench/__main__.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/python3
+#
+# Copyright 2020 Google LLC
```

Comment:
Thanks for the thorough scrub and move! 
Just one nit and q: Did we ever decide on keeping the original date or updating to 2021 for changes like these?

Reply:
This is a mistake, fixed.